sdk/keyvault/keyvault-keys/src/localCryptography/hash.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createHash as cryptoCreateHash } from "crypto";
+import { isNode } from "@azure/core-http";
+
+/**
+ * @internal
+ * @ignore
+ * Use the platform-local hashing functionality
+ */
+export async function createHash(algorithm: string, data: Uint8Array): Promise<Buffer> {
+  if (isNode) {
+    const hash = cryptoCreateHash(algorithm);
+    hash.update(Buffer.from(data));
+    const digest = hash.digest();
+    return digest;
+  } else {
+    if (window && window.crypto && window.crypto.subtle) {
+      return Buffer.from(await window.crypto.subtle.digest(algorithm, Buffer.from(data)));
```

Comment:
if we're in the browser, where is `Buffer` coming from?

Reply:
Great question. I'm not sure how I missed this before. Thank you. I will investigate if this isn't working or what.

Reply:
This doesn't work as expected. For a bit I was freaking out, but the KeyVault client has never actually worked in the browsers because of the lack of CORS support. Should I drop browser support here too?

We should talk about making a separate project inside of this repo to tackle down all of our cryptography needs, outside of KeyVault. I really believe it needs to be a core product we support. I have tried to push for this idea before, but so far I have failed.

Reply:
I'm provisionally ok with a new `core-crypto` package that abstracts Node/window.crypto differences

Reply:
Sounds good. This will take time though. Ramya and the rest of the team is aware of this. For now I'll drop browser support in this method (granted, none of the keyvault clients can be used from the browser right now)